tests/ui/lifetimes/issue-107492-default-value-for-lifetime.rs

```diff
@@ -1,5 +1,6 @@
 pub struct DefaultLifetime<'a, 'b = 'static> {
                                    //~^ ERROR unexpected default lifetime parameter
+                                   //~^^ ERROR expected one of `,`, `:`, or `>`, found `=`
```

Comment:
```suggestion
                                   //~| ERROR expected one of `,`, `:`, or `>`, found `=`
```

nit: this chaining style is preferred, but once you apply the above suggestion you probably will only have one error.